src/task_list/moduls.py

```diff
@@ -0,0 +1,146 @@
+import json
+import pathlib
+
+
+class DataJson:
+    'Класс для работы с файлом'
+
+    def __init__(self):
+        self.file = pathlib.Path('task_list/src/config/data.json').resolve()
+
+
+    def read_data_json(self):    # Чтение файла с задачами
```

Comment:
оценка пункт 2

```suggestion
    def read_data_json(self) -> dict[str, Any]:
        """Чтение файла с задачами """    
```

добавляем типизацию, комментарии пишутся под функцией